roc/log.c

```diff
@@ -3,4 +3,4 @@
 
 void rocGoLogHandlerProxy(const roc_log_message* message, void* argument) {
     rocGoLogHandler((roc_log_message*)message);
-}
+}
```

Comment:
Please rollback this change.

Reply:
what change?

Reply:
Huh, github does not show it in all views. If you open https://github.com/roc-streaming/roc-go/pull/80/files, you can see it:

![image](https://github.com/roc-streaming/roc-go/assets/8436629/5872725c-edcd-421b-9fa1-b8c961495684)

It means that `\n` was removed from the end of the file, in other words, the last line in the file now does not have trailing newline, unlike all other lines. Probably your editor did that.

Reply:
> Sorry for asking this again, but did you self-review and test your code before submitting it? I mean, enable logging and actually verify the logger output?
> 
> Formatting is still broken; the messages produced by logWrite are a mess.
> 
> This is a bit of wasting time of both of us, because I need to debug it locally, write a comment, then you need to read my comment, and debug it locally. So we're doubling debugging work and spend extra time for communication.

I'm sorry for this, but can i ask how i can enable logging and actually test it on my end... If there's a specific way that you do this